### PR TITLE
Add sigmoid at test time for stackex example

### PR DESCRIPTION
**Pull request by toenshoff**
This PR makes a small change to the GNN example script to apply sigmoid activation to the outputs when testing.

When running for binary classification tasks, the previous version passes raw logits to `task.evaluate`, but the metrics seem to expect sigmoid-activated outputs as they threshold at 0.5.

### CODE REVIEW

**Comment by weihua916** [comment]
Indeed! https://github.com/snap-stanford/relbench/blob/main/relbench/metrics.py#L12
Great catch!

**Review by weihua916 — APPROVED** [review]
